test/Unit/Durable/DurableTimerTests.cs

```diff
@@ -0,0 +1,260 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+namespace Microsoft.Azure.Functions.PowerShellWorker.Test.Durable
+{
+    using System;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System.Linq;
+    using System.Management.Automation;
+    using System.Threading;
+    using Microsoft.Azure.Functions.PowerShellWorker.Durable;
+    using Moq;
+    using Xunit;
+
+    public class DurableTimerTests
+    {
+        private const DateTimeKind utc = DateTimeKind.Utc;
```

Comment:
I don't think this constant serves any useful purpose, let's use DateTimeKind.Utc directly.